run.go

```diff
@@ -115,33 +115,33 @@ type Run struct {
 
 // RunActions represents the run actions.
 type RunActions struct {
-	IsCancelable      bool `json:"is-cancelable"`
-	IsConfirmable     bool `json:"is-confirmable"`
-	IsDiscardable     bool `json:"is-discardable"`
-	IsForceCancelable bool `json:"is-force-cancelable"`
+	IsCancelable      bool `jsonapi:"attr,is-cancelable"`
+	IsConfirmable     bool `jsonapi:"attr,is-confirmable"`
+	IsDiscardable     bool `jsonapi:"attr,is-discardable"`
+	IsForceCancelable bool `jsonapi:"attr,is-force-cancelable"`
 }
 
 // RunPermissions represents the run permissions.
 type RunPermissions struct {
-	CanApply        bool `json:"can-apply"`
-	CanCancel       bool `json:"can-cancel"`
-	CanDiscard      bool `json:"can-discard"`
-	CanForceCancel  bool `json:"can-force-cancel"`
-	CanForceExecute bool `json:"can-force-execute"`
+	CanApply        bool `jsonapi:"attr,can-apply"`
+	CanCancel       bool `jsonapi:"attr,can-cancel"`
+	CanDiscard      bool `jsonapi:"attr,can-discard"`
+	CanForceCancel  bool `jsonapi:"attr,can-force-cancel"`
+	CanForceExecute bool `jsonapi:"attr,can-force-execute"`
 }
 
 // RunStatusTimestamps holds the timestamps for individual run statuses.
 type RunStatusTimestamps struct {
-	ErroredAt            time.Time `json:"errored-at"`
-	FinishedAt           time.Time `json:"finished-at"`
-	QueuedAt             time.Time `json:"queued-at"`
-	StartedAt            time.Time `json:"started-at"`
-	ApplyingAt           time.Time `json:"applying-at"`
-	AppliedAt            time.Time `json:"applied-at"`
-	PlanningAt           time.Time `json:"planning-at"`
-	PlannedAt            time.Time `json:"planned-at"`
-	PlannedAndFinishedAt time.Time `json:"planned-and-finished-at"`
-	PlanQueuabledAt      time.Time `json:"plan-queueable-at"`
+	ErroredAt            string `jsonapi:"attr,errored-at"`
+	FinishedAt           string `jsonapi:"attr,finished-at"`
+	QueuedAt             string `jsonapi:"attr,queued-at"`
+	StartedAt            string `jsonapi:"attr,started-at"`
+	ApplyingAt           string `jsonapi:"attr,applying-at"`
+	AppliedAt            string `jsonapi:"attr,applied-at"`
+	PlanningAt           string `jsonapi:"attr,planning-at"`
+	PlannedAt            string `jsonapi:"attr,planned-at"`
+	PlannedAndFinishedAt string `jsonapi:"attr,planned-and-finished-at"`
+	PlanQueuabledAt      string `jsonapi:"attr,plan-queueable-at"`
```

Comment:
This is fixed [here](https://github.com/hashicorp/jsonapi/pull/1), will update this comment soon.